### DISCUSSION TITLE
health:list with --fail-command-on-failing-check option

### DISCUSSION THREAD

**Discussion by eleftrik** [spatie/laravel-health#74] [Ideas]
In my opinion, `health:list` command should have `--fail-command-on-failing-check` option like `health:check`.

I'm using `health:list --fresh` in a script and I need to check the exit code to see if there is some error/warning.
I know in my script I could use `php artisan health:check --fail-command-on-failing-check` (which returns an appropriate exit code), but `health:list` output is cooler 😎

Thanks

**Comment by freekmurze** [comment]
Feel free to add that. Make sure to included tests and update the readme 👍

**Reply by eleftrik** [reply]
@freekmurze Just submitted a small PR (my first ever, so please be kind 😅).
Thanks for your wonderful packages and your great support to the Laravel community!
